.circleci/config.yml

```diff
@@ -403,6 +409,7 @@ workflows:
             branches:
               only:
                 - master
+                - 667-everything-is-so-broken
```

Comment:
This still needed for the merge?

Reply:
Should be removed before merge; but I want to get E2E working before removing this.